src/main/java/me/champeau/gradle/japicmp/ignore/CompatibilityChangesFilter.java

```diff
@@ -0,0 +1,143 @@
+package me.champeau.gradle.japicmp.ignore;
+
+
+import japicmp.model.JApiClass;
+import me.champeau.gradle.japicmp.archive.Diff;
+import me.champeau.gradle.japicmp.ignore.element.Element;
+import me.champeau.gradle.japicmp.ignore.entity.ChangeEntity;
+import me.champeau.gradle.japicmp.ignore.entity.Entity;
+import me.champeau.gradle.japicmp.ignore.entity.EntityManager;
+import me.champeau.gradle.japicmp.ignore.entity.RemoveEntity;
+import me.champeau.gradle.japicmp.ignore.processor.ClassProcessor;
+import org.yaml.snakeyaml.Yaml;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class CompatibilityChangesFilter {
+  private final List<DeclaredModule> declaredModules;
+
+  @SuppressWarnings("unchecked")
+  public CompatibilityChangesFilter(File configFile) {
+    Yaml yaml = new Yaml();
+    HashMap<String, List<Map<String, String>>> objects = null;
+    try {
+      objects = yaml.loadAs(new FileInputStream(configFile), HashMap.class);
+    } catch (FileNotFoundException e) {
+      e.printStackTrace();
+    }
+
+    declaredModules = fillProjects(objects);
+  }
+
+  private List<DeclaredModule> fillProjects(HashMap<String, List<Map<String, String>>> objects) {
+    List<DeclaredModule> result = new ArrayList<>();
+    if (objects != null) {
+      for (Map.Entry<String, List<Map<String, String>>> projectEntry : objects.entrySet()) {
+        result.add(parseAsModule(projectEntry));
+      }
+    }
+    return result;
+  }
+
+  private DeclaredModule parseAsModule(Map.Entry<String, List<Map<String, String>>> projectEntry) {
```

Comment:
Meaningful parameters look more conceivable
```suggestion
private DeclaredModule parseAsModule(String name, List<Map<String, String>> values) {
```

Reply:
Done